lib/vector2.dart

```diff
@@ -0,0 +1,88 @@
+library flame.vector_math;
+
+export 'package:vector_math/vector_math_64.dart' show Vector2;
+
+import 'dart:math';
+import 'dart:ui';
+import 'package:vector_math/vector_math_64.dart';
+
+class Vector2Operations {
+  /// Creates converting integers to double.
+  ///
+  /// Internal representation is still using double, the conversion is made in the constructor only.
+  static Vector2 fromInts(int x, int y) => Vector2(x.toDouble(), y.toDouble());
+
+  /// Creates a [Vector2] using an [Offset]
+  static Vector2 fromOffset(Offset offset) => Vector2(offset.dx, offset.dy);
+
+  /// Creates a [Vector2] using an [Size]
+  static Vector2 fromSize(Size size) => Vector2(size.width, size.height);
+
+  /// Creates a [Vector2] using a [math.Point]
+  static Vector2 fromPoint(Point point) => Vector2(point.x, point.y);
+
+  /// Creates an [Offset] from a [Vector2]
+  static Offset toOffset(Vector2 v) => Offset(v.x, v.y);
```

Comment:
can these be extension functions?

Reply:
Sadly not, since the constructors are using factories.